Kernel/Ordering.hpp

```diff
@@ -103,6 +108,8 @@ class Ordering
 
   Result compareEqualities(Literal* eq1, Literal* eq2) const;
 
+  static bool containsVar(const AppliedTerm& s, TermList var);
```

Comment:
`unsigned var` so you can't pass a `Term *`?

Reply:
This would mean having three `TermList` ctors inside the function, but sure. I just thought it's easier if we add an assertion.

Reply:
Up to you.